src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,102 @@
 package core.basesyntax;
 
+import java.util.LinkedList;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private int initialCapacity = 16;
+    private float loadFactor = 0.75f;
```

Comment:
The name `loadFactor` implies that it's a constant, but it's not declared as one. According to the checklist, any magic numbers should be constants. Please declare `loadFactor` as a constant field.